docs/en/04_Changelogs/4.11.0.md

```diff
@@ -7,9 +7,10 @@ title: 4.11.0 (unreleased)
 ## Overview
 
 - [Regression test and Security audit](#audit)
-- [Dropping support for PHP 7.3](#phpeol)
 - [GraphQL v4 major release](#graphqlv4)
 - [Features and enhancements](#features-and-enhancements)
+  - [Adding support for PHP 8.1](#php81)
```

Comment:
I don't think the two PHP things should live under "features and enhancements", those are meant for more regular things.

PHP 8.1 support is a headline feature, along with graphl4, and dropping PHP7.3 is arguably more relevant than both, and it's certainly not a "feature or enhancement"

We had them on the top level in the beta changelog - https://github.com/silverstripe/silverstripe-framework/blob/4/docs/en/04_Changelogs/beta/4.11.0-beta1.md#php81

Reply:
I think I agree with you - I'll give other people a chance to weigh in if they want to (there's no real urgency on getting this merged and there's quite a bit more work to be done on other acceptance criteria for its parent issue) before making the change but most likely will move it back above "Features and enhancements".

Reply:
I've moved these back.